cli/src/migrations/files.ts

```diff
@@ -50,6 +50,7 @@ export async function getLocalMigrationFiles(): Promise<Schemas.MigrationObject[
   const migrations: Schemas.MigrationObject[] = [];
 
   for (const entry of ledger) {
+    if (entry === '') continue;
```

Comment:
This is a fix for `newlines` that were parsed incorrectly as migration files, not directly related to this PR but I noticed while testing.

Reply:
@SferaDev I would just add a comment explaining, because I definitely wouldn't understand it from the code.